### PR TITLE
Remove extra blank lines

### PR DESCRIPTION
**Pull request by p8**
### Motivation / Background

This removes some arbitrary double blank lines.
We should probably have some RuboCop rules for this.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

**Comment by zzak** [comment]
There is a markdownlint rule for [this](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md012---multiple-consecutive-blank-lines) and remember we discussed turning it off in the original PR because it was a lot of fruitless changes. The resulting output is the same.

In this case, it looks like there might be a rubocop [rule](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/EmptyLines) but we should try to enable that first to see the impact.

Essentially my take is, if we are going to make cosmetic changes there should be a linter that enforces it otherwise someone has to monitor them, or we end up with several of the same PRs for places we missed which all creates unnecessary workload for us.

**Comment by zzak** [comment]
>  if we are going to make cosmetic changes there should be a linter that enforces it otherwise someone has to monitor them, or we end up with several of the same PRs for places we missed which all creates unnecessary workload for us.

Probably should mention this is _my_ opinion, and not the decision of any committers or core member, so take it with a grain of salt. 🧂 

That said, I've opened #47854 to see the impact of enforcing this globally and included the autocorrections. Feel free to merge or decline it!

File: activerecord/lib/active_record/insert_all.rb [hunk 1]
```diff
@@ -57,7 +57,6 @@ def primary_keys
       Array(connection.schema_cache.primary_keys(model.table_name))
     end
 
-
```
Comment (kamipo) [inline activerecord/lib/active_record/insert_all.rb:60]: Previously Basecamp guys said these newlines were intentional for method grouping.

https://github.com/rails/rails/pull/38899#discussion_r414679692

**Comment by p8** [comment]
Ok closing this in favor of: https://github.com/rails/rails/pull/47854

**Pull request closed without merging**